modules/giapi/src/test/scala/giapi/client/GiapiStatusSpec.scala

```diff
@@ -134,8 +127,7 @@ final class GiapiStatusSpec extends CatsSuite {
         GmpStatus.closeGmpStatus(g) >> c.get[Int](intItemName)
       }
       .attempt
-      .unsafeRunSync() should matchPattern { case Left(GiapiException(_)) =>
-    }
+      .map(matchPattern { case Left(GiapiException(_)) => })
```

Comment:
I don't think munit includes this. Is this still on scalatest?

Reply:
Yes, the source code imports just that one matcher.